$NetBSD: patch-CMakeLists.txt,v 1.4 2022/10/21 16:10:37 nros Exp $

* respect LIB_SUFFIX set to an empty string, so
  that LIB_SUFFIX can be set to nothing for pkgsrc

--- CMakeLists.txt.orig	2022-10-20 16:44:38.408278688 +0000
+++ CMakeLists.txt
@@ -101,7 +101,7 @@ include(CheckLibraryExists)
 include(CheckIncludeFiles)
 include(CheckFunctionExists)
 
-if (NOT LIB_SUFFIX)
+if (NOT DEFINED LIB_SUFFIX)
     set(LIBDIRNAME "lib")
     # this caused problems in debian where it has to always be lib....
     if (NOT EXISTS /etc/debian_version)
